apps/browser/src/content/sites/_default/types.ts

```
@@ -1,14 +1,9 @@
-	MessageType,
-	BrowserObj,
-} from '@eurora/browser-shared/content/extensions/watchers/watcher';
+import type { MessageType, BrowserObj } from '../../../shared/content/extensions/watchers/watcher';
 
 export type ArticleMessageType = MessageType;
 
 // eslint-disable-next-line @typescript-eslint/no-empty-object-type
-export interface WatcherParams {
-	// Article watcher doesn't need specific parameters for now
-}
+export interface WatcherParams {}
```

Biome (2.3.14)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)